renderer.go

```diff
@@ -33,7 +33,7 @@ func SetDefaultRenderer(r *Renderer) {
 
 // NewRenderer creates a new Renderer.
 //
-// w will be used to determine the terminal's color capabilities.
+// It will be used to determine the terminal's color capabilities.
```

Comment:
`w` is actually referring to the `io.Writer` that gets passed in. As such it's not a typo.

Reply:
Ah, my mistake!